.github/workflows/gh_pages.yml

```diff
@@ -1,30 +1,30 @@
 name: Github Pages
 on:
-  release:
-    types: [published]
+  push:
+    branches:
+      - master
 jobs:
   build-and-deploy:
     runs-on: ubuntu-latest
     steps:
-      - name: Checkout
```

Comment:
Any reason for removing the step name?

Reply:
Hm, for some reason, I thought that if you look at a [particular build breakdown](https://github.com/stellar/js-stellar-base/runs/3437811362?check_suite_focus=true) and opened the "Checkout" tab, there were no results. I figured that made it makes sense to drop the name since there would never be a reason to look at it, but now I see that there *are* logs... maybe they were slow to load, or something. I'll put it back!